packages/ui/package.json

```diff
@@ -0,0 +1,80 @@
+{
+	"name": "@automattic/ui",
+	"version": "1.0.0",
+	"description": "Automattic Design System UI components.",
+	"homepage": "https://github.com/Automattic/wp-calypso",
+	"license": "GPL-2.0-or-later",
+	"author": "Automattic Inc.",
+	"main": "dist/index.js",
+	"module": "dist/index.mjs",
+	"calypso:src": "src/index.ts",
+	"exports": {
+		".": {
+			"calypso:src": "./src/index.ts",
+			"import": "./dist/index.mjs",
+			"require": "./dist/index.js"
+		},
+		"./style.css": {
+			"import": "./dist/index.css",
+			"require": "./dist/index.css"
+		},
+		"./src/*": {
+			"calypso:src": "./src/*"
+		}
+	},
+	"sideEffects": [
+		"*.css",
+		"*.scss"
+	],
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/Automattic/wp-calypso.git",
+		"directory": "packages/ui"
+	},
+	"publishConfig": {
+		"access": "public"
+	},
+	"bugs": "https://github.com/Automattic/wp-calypso/issues",
+	"files": [
+		"dist",
+		"src"
+	],
+	"types": "dist",
+	"dependencies": {
+		"@wordpress/base-styles": "5.23.0",
+		"@wordpress/compose": "7.23.0",
+		"@wordpress/element": "^6.23.0",
+		"@wordpress/icons": "^10.23.0",
+		"@wordpress/primitives": "^4.23.0",
```

Comment:
The lack of `@wordpress/element` and `@wordpress/primitives` here in my initial build attempt caused the `Icon` component to break when I tried to use it in a test project, due to it not being able to do the `icon.type === SVG` check correctly. This kind of problem is hard to catch, because it works fine in calypso/storybook.

https://github.com/Automattic/wp-calypso/blob/d5c0e7c1d8bc09bb81e709561f7b3275b237af9d/packages/components/src/icon/index.tsx#L56

Punted to follow-up: I think we should enforce the `import/no-extraneous-dependencies` lint rule, at least in this package. The TS checking was not sufficient in this case.